client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/index.tsx

```diff
@@ -465,7 +465,8 @@ const PatternAssembler = ( {
 	};
 
 	const stepContent = (
-		<div
+		<NavigatorProvider
```

Comment:
The large preview on the right side needs to access the navigator so it can open the header screen programmatically